src/Manager.php

```diff
@@ -534,8 +534,12 @@ public function getMember()
      */
     public function getQueryFromPersistedID($id)
     {
-        /** @var PersistedQueryMappingProvider $provider */
-        $provider = Injector::inst()->create(PersistedQueryMappingProvider::class);
+        try {
+            /** @var PersistedQueryMappingProvider $provider */
+            $provider = Injector::inst()->get(PersistedQueryMappingProvider::class);
```

Comment:
This should be `get` instead of `create` to prevent the Injector from creating instances every time. Now the Injector will find or create the singleton of the Provider, and it fixed the failure in the unit test.